universe/vikram.hello/manifest.tt

```diff
@@ -0,0 +1,102 @@
+                                        base = ["date", "release date", "release year", "release day"],
+                                        property=["release date #"],
+                                        preposition=["from"],
+                                        passive_verb=["released in #", "filmed #", "released #"],
+                                        adjective_argmin = ["least recent", "oldest", "first"],
+                                        adjective_argmax = ["most recent", "latest", "newest", "now playing"],
+                                        passive_verb_argmax=["released most recently"],
+                                        passive_verb_projection=['released | on', 'filmed | on'],
+                                    }],
+                                    out rating : Enum(G, PG, PG13, R)
+                                    #_[canonical = {
+                                        default="base",
+                                        base=["rating"],
+                                        passive_verb=["rated #"],
+                                        preposition = ["with # rating"],
+                                        adjective_enum = {
+                                            G = ["suitable for children", "mild"],
+                                            R = ["violent", "bloody", "made for adults"]
+                                        }
+                                    }],
+                                    out rating_score : Number
+                                    #[min_number=0]
+                                    #[max_number=10]
+                                    #_[canonical = {
+                                        default = "base",
+                                        base = ["rating", "score"],
+                                        adjective_argmin = ["worst"],
+                                        adjective_argmax = ["best"]
+                                    }],
+                                    out actors : Array(Entity(vikram.hello:person))
+                                    #_[canonical = {
+                                        default = "preposition", 
+                                        base = ["actor", "actress"],
+                                        property = ["actor #", "actress #"],
+                                        preposition = ["by #", "with #"],
+                                        passive_verb = ["acted by #", "played by #", "acted with #"],
+                                        base_projection = ['actor', 'actress'],
+                                        passive_verb_projection = ['acted | by', 'played | by']
+                                    }],
+                                    out poster: Entity(tt:picture)
+                                    #[filterable = false]
+                                    #_[canonical="poster"])  
+    #_[canonical = ["movie"]]
```

Comment:
You might want to add more synonyms of movie here: "film", "flick", maybe others?